scripts/gdb/php_gdb.py

```diff
@@ -961,6 +961,9 @@ def array_size(ary_type):
     return ary_type.fields()[0].type.range()[1]+1
 
 def format_zstr(zstr):
+    if zstr.type.code == gdb.TYPE_CODE_PTR and int(zstr) == 0:
+        return zstr
```

Comment:
This change is not necessary if we remove these lines from `ZendAstPrettyPrinter.children`:

```
            elif field.name == 'name':
                yield (field.name, format_zstr(val[field.name]))
```

This field will be formatted by `format_nested()`, which handles the null case.